CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing 
+Thanks for helping us make the HPE Design System better! You came to the right place to start your contribution.
+
+## Project Structure
+The HPE Design System is a monorepo built from two modules.
+See the [README](https://github.com/hpe-design/design-system#welcome-to-hpe-design-system)
+for a break down of the Design System structure.
+
+## How to Contribute 
+Code, Code, Code...
```

Comment:
Recommend removing this section for now.

Reply:
okay sounds good!